lib/quby/questionnaires/dsl/charting/chart_builder.rb

```diff
@@ -52,6 +56,13 @@ def build(&block)
         end
 
         def validate!
+          if @chart.y_categories.present? && @chart.y_range.present?
+            if @chart.y_range != (0..(@chart.y_categories.count - 1))
+              fail ArgumentError, 'Y_categories size and range do not match'
+            end
+          elsif @chart.y_categories.present? && @chart.y_range.blank?
+            @chart.y_range = (0..(@chart.y_categories.count - 1))
```

Comment:
validate method met een bijwerking is niet ok. 
In de chart entity een getter definieren?